webapp/store/snap_details_views.py

```diff
@@ -77,6 +80,27 @@ def _get_context_snap_details(snap_name):
 
         icons = logic.get_icon(details["snap"]["media"])
 
+        publisher_info_and_snaps = helpers._get_file(
+            "/{}{}.yaml".format(
+                flask.current_app.config["CONTENT_DIRECTORY"][
+                    "PUBLISHER_PAGES"
+                ],
+                details["snap"]["publisher"]["username"],
+                ".yaml",
```

Comment:
You can remove this line.

Reply:
This is still relevant, no? This file hasn't been changed?